lib/pbench/server/auth/auth.py

```diff
@@ -168,13 +250,13 @@ def verify_third_party_token(auth_token: str, oidc_client: OpenIDClient) -> bool
 
         if not oidc_client.TOKENINFO_ENDPOINT:
             Auth.logger.warning("Can not perform OIDC online token verification")
-            return False
+            return None
 
         try:
             token_payload = oidc_client.token_introspect_online(
                 token=auth_token, token_info_uri=oidc_client.TOKENINFO_ENDPOINT
             )
         except OpenIDClientError:
-            return False
+            return None
```

Comment:
Same question here about the function return value...

Reply:
Fixed.